src/test/java/io/supertokens/test/TelemetryAPITest2_7.java

```diff
@@ -39,4 +48,97 @@ public void beforeEach() {
         Utils.reset();
     }
 
+    @Test
+    public void testTelemetryDisabledInMemDb() throws Exception {
+        String[] args = {"../"};
+
+        TestingProcessManager.TestingProcess process = TestingProcessManager.start(args);
+        assertNotNull(process.checkOrWaitForEvent(ProcessState.PROCESS_STATE.STARTED));
+
+        if (!Version.getVersion(process.getProcess()).getPluginName().equals("sqlite")) {
+            return;
+        }
+
+        JsonObject response = io.supertokens.test.httpRequest.HttpRequest
+                .sendGETRequest(process.getProcess(), "",
+                        "http://localhost:3567/telemetry", new HashMap<>(), 1000,
+                        1000,
+                        null, Utils.getCdiVersion2_7ForTests(), ThirdParty.RECIPE_ID);
```

Comment:
Recipe ID should be "" for all tests in this file

Reply:
Alright making the change